### DISCUSSION TITLE
Lazily Rendered Tabs (with `keepMounted=true`)

### DISCUSSION THREAD

**Discussion by tony19** [mantinedev/mantine#4488] [Feature requests]
The `Tabs` component renders all tabs eagerly by default, and I need them to be rendered lazily. That is, I need the tabs to render upon first opening the tab and **stay mounted when I switch tabs**. It would also be interesting to enable this behavior for specified tabs, where only a subset of tabs are rendered lazily.

`keepMounted=false` doesn't fit my use case because it makes the tabs always unmount and re-render.

I can implement a workaround with my own flags, but I think it would be nicer for Mantine to do this out of the box.

Can we add a property to enable the behavior I described?

**Comment by jaguardo** [comment]
From the documentation, this sounds like the intended way things should work.
`keepMounted` would imply that the tab stays mounted, even when the tab is changed... but that doesn't seem to be the case.
What is the workaround you created?
